code/renderers/svelte/src/public-types.test.ts

```diff
@@ -172,11 +172,11 @@ describe('Story args can be inferred', () => {
     expectTypeOf(Basic).toEqualTypeOf<Expected>();
   });
 
-  const withDecorator: DecoratorFn<{ decoratorArg: string }> = (
+  const withDecorator: Decorator<{ decoratorArg: string }> = (
     storyFn,
     { args: { decoratorArg } }
```

Comment:
How would things look like if the decorator had globals?

Reply:
I guess we should allow module augmentation for globals as well :)